packages/block-editor/src/hooks/anchor.js

```diff
@@ -71,7 +71,18 @@ export const withInspectorControl = createHigherOrderComponent( ( BlockEdit ) =>
 					<InspectorAdvancedControls>
 						<TextControl
 							label={ __( 'HTML Anchor' ) }
-							help={ __( 'Anchors lets you link directly to a section on a page.' ) }
+							help={ (
+								<>
+									<p>
+										{ __( 'Enter a word or two — without spaces — to make a unique web address just for this heading, called an “anchor.” Then, you’ll be able to link directly to this section of your page.' ) }
+									</p>
+									<p>
```

Comment:
In any case, if we stick to using `<p>` instead of CSS, only the first `<p>` is needed so the one wrapping `<ExternalLink>` can be removed.